### DISCUSSION TITLE
Can we use CSS variables in use-media-query?

### DISCUSSION THREAD

**Discussion by Toshinaki** [mantinedev/mantine#5320] [Q&A]
The default CSS variable `--mantine-breakpoint-md` is `62em`.

Is there any way use that variable in `useMediaQuery`?
```
const matches = useMediaQuery('(min-width: 56.25em)');
```
to something like:
```
const matches = useMediaQuery('(min-width: --mantine-breakpoint-md)');
```

**Comment by rtivital** [comment]
No, CSS variables cannot be used in media queries, neither in css nor in js files.
